target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -136,6 +136,9 @@ ramips_setup_interfaces()
 	engenius,esr600)
 		ucidef_add_switch "switch0" \
 			"1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1" "5:wan" "0@eth0"
+		ucidef_add_switch "switch1" \
+			"0:lan" "1:lan" "2:lan" "3:lan" "6@eth0"
```

Comment:
If you really want a fake topology, at least have the port used to connect the external switch here instead of such a random port choice.